meson.build

```diff
@@ -59,6 +59,8 @@ include_default = '''
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <assert.h>     # For static_assert
```

Comment:
`# For static_assert` should be `/* # For static_assert */` otherwise a warning of extra token is thrown. Same for the other include

Reply:
Thanks! Hmm, interesting that it worked at all.